mysql-test/suite/innodb/t/temp_table_encrypt.test

```diff
@@ -94,4 +94,23 @@ INSERT INTO t01 VALUES (1), (2), (3);
 SET GLOBAL innodb_temp_tablespace_encrypt = OFF;
```

Comment:
Now redundant with the restart

Reply:
but this is after restart, at the end of test right?

Reply:
never mind, this is at other place